client/src/components/Map/infoWindow/currLocation.js

```diff
@@ -142,24 +145,22 @@ class InfoWindow extends React.Component {
               </a>
             )}
             {userPlayer && (
-              <Player
-                item={this.state.item}
-                is_playing={this.state.is_playing}
-                progress_ms={this.state.progress_ms}
-                is_ad={this.state.is_ad}
-                is_me={true}
+              <SpotifyPlayer
+                uri={uri}
+                size="large"
+                view="coverart"
+                theme="black"
               />
             )}
             {otherPlayer && (
-              <Player
-                item={this.state.item}
-                is_playing={this.state.is_playing}
-                progress_ms={this.state.progress_ms}
-                is_ad={this.state.is_ad}
-                is_me={false}
+              <SpotifyPlayer
```

Comment:
I think we can reduce this code by having `{(userPlayer || otherPlayer) && ...` in line 148

Reply:
Forgot to add, maybe "compact" in `size` would work so we don't have big songInfo cards